templates/Default/engine/Default/alliance_stat.php

```diff
@@ -16,12 +16,32 @@
 
 if ($CanChangeChatChannel) { ?>
 	<tr>
-		<td class="top">Discord Channel ID:&nbsp;</td>
-		<td><input type="text" name="discord" size="30" value="<?php echo htmlspecialchars($Alliance->getDiscordChannel()); ?>" /></td>
+		<td class="top">Discord Server ID:&nbsp;</td>
+		<td>
+			<input type="text" name="discord_server" size="30" value="<?php echo htmlspecialchars($Alliance->getDiscordServer()); ?>" />
+			<br />Adds the Discord join server widget to the Message of the Day page.
```

Comment:
Could you add a little info on where you get the server url? I'm guessing that it'd be via the invite link or similar but I'm actually not sure, so I expect there'll be a bunch of questions about it

Reply:
Sure, do you prefer it to link to an SMR Wiki section about it or to include it in the page?

You go to `Server Settings -> Widget` and then there's a `Server ID` field with a "Copy" button. All you need is the ID, not a URL. Then on the Discord side you have to click "Enable Server Widget" and if you also want it to display a "Connect" button on the widget you need to set an "Instant Invite Channel" from the dropdown menu.

I guess one benefit of a wiki page is that it could provide some image examples.

Reply:
A wiki page probably makes sense, it'll encourage more wiki use and it allows things like image examples and easy updating